proof/invariant-abstract/ARM/ArchCSpace_AI.thy

```diff
@@ -543,7 +543,7 @@ lemma is_cap_simps':
   "is_ntfn_cap cap = (\<exists>r b R. cap = cap.NotificationCap r b R)"
   "is_zombie cap = (\<exists>r b n. cap = cap.Zombie r b n)"
   "is_arch_cap cap = (\<exists>a. cap = cap.ArchObjectCap a)"
-  "is_reply_cap cap = (\<exists>x R. cap = cap.ReplyCap x R)"
+  "is_reply_cap cap = (\<exists>x. cap = cap.ReplyCap x)"
```

Comment:
While we're here, would a more informative name for the green variable `x` be nicer?